### ISSUE TITLE
Volume in volume mixer keeps lowering

### ISSUE THREAD

**Issue by MuffinCrow** [ottomated/CrewLink#68]
Randomly, the volume mixer will lower the volume of CrewLink and I can't find a way to stop it.

**Comment by SkittlestheButt** [comment]
Yeah this happens to me as well. It also keeps lowering discord volume in the mixer. Which isn't as bad since I don't use it while using CrewLink, but its confusing. I'm thinking it only really does that when you mess with the CrewLink settings but I'm not really sure.

**Comment by pau101** [comment]
You can try right clicking "Speakers / Headphones" in the task bar then click "Sounds" and go the "Communications" tab, then select "Do nothing"

**Comment by Chronove** [comment]
Can confirm, settings are as @pau101 states. It lowers volume of CrewLink, Discord and sometimes Chrome to like 13% at random
